packages/marshal/marshal.js

```diff
@@ -29,7 +29,7 @@ const remotableToInterface = new WeakMap();
 
 /**
  * Simple semantics, just tell what interface (or undefined) a remotable has.
- *
+ * @callback GetInterfaceOf
  * @param {*} maybeRemotable the value to check
  * @returns {InterfaceSpec|undefined} the interface specification, or undefined
  * if not a Remotable
```

Comment:
I don't think you can use `@callback` to explicitly name a function type and at the same time have it apply to the following function. (@michaelfig please correct if I have that wrong.) In any case, it's not how we've been doing it. Rather

```suggestion
 * @callback GetInterfaceOf
 * @param {*} maybeRemotable the value to check
 * @returns {InterfaceSpec|undefined} the interface specification, or undefined
 * if not a Remotable
 */

/** @type {GetInterfaceOf} */
```

Reply:
Good call. The examples seem to indicate that https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#typedef-callback-and-param

Reply:
Done